README.md

```diff
@@ -22,6 +18,24 @@ ddev add-on get ddev/ddev-adminer && ddev restart
 
 Then you can just `ddev adminer` or use `ddev describe` to get the URL (`https://<project>.ddev.site:9101`).
 
+To customize Adminer setup:
+
+```sh
+# drivers: server, sqlite, sqlite2, pgsql, oracle, mssql, mongo, and more via plugins
+# plugins: https://www.adminer.org/en/plugins/
+# design: https://www.adminer.org/en/#extras
+ddev dotenv set .ddev/.env.adminer \
+    --adminer-docker-image="adminer:standalone" \
+    --adminer-default-driver=server \
+    --adminer-default-db=db \
+    --adminer-default-username=db \
+    --adminer-default-password=db \
+    --adminer-plugins="tables-filter edit-calendar" \
+    --adminer-design=dracula
```

Comment:
I think this section about customization should be clearly marked "for advanced users, not required".  I don't think the default db/username/password are going to help anybody. They will just get in trouble. 

So then that doesn't leave very much to config, unless somebody wants to change the image. 

Anyway, try to make it so people don't actually try to do this unless they're seriously advanced and have a tremendous need.

(Or this will end up like our global setting `xdebug_ide_location` that benefits almost nobody but from time to time sends the wrong people down a complete rabbit-hole.)